frontend/express/public/core/events/javascripts/countly.details.views.js

```diff
@@ -107,19 +300,48 @@ var allEventsVuex = [{
     clyModel: countlyAllEvents
 }];
 
+var compareEventsVuex = [{
+    clyModel: countlyCompareEvents
+}];
+
 var AllEventsViewWrapper = new countlyVue.views.BackboneWrapper({
-    component: AllEventsView,
+    component: eventsWrapper,
     vuex: allEventsVuex,
     templates: [
-        "/core/events/templates/allEvents.html"
+        "/core/events/templates/allEvents.html",
+        "/core/events/templates/compareEvents.html"
+    ]
+});
+
+var CompareEventsViewWrapper = new countlyVue.views.BackboneWrapper({
```

Comment:
please initialize the backbone when the route is triggered.
follow this pattern - https://github.com/Countly/countly-server/blob/9b1cfc1d58e7df216e5497bf2a47f3b46bd8a434/plugins/vue-example/frontend/public/javascripts/countly.views.js#L737-L759